cypress/integration/about/status.spec.js

```diff
@@ -0,0 +1,24 @@
+/// <reference types="cypress" />
+import {loadAndAcceptDisclaimer} from "../../support/index"
+
+const ENDPOINTS = ['Animal & Veterinary › Adverse Events', 'Devices › Classification',
```

Comment:
Is there any way we can get around adding these individually? It's doable if needed, but I'd rather avoid adding another area to change when adding a new endpoint if possible.